packages/core/scripts/gas-rebate/index.js

```diff
@@ -0,0 +1,301 @@
+const cliProgress = require("cli-progress");
```

Comment:
Can you please add a comment explaining what this script does and how to run it up at the top?

Reply:
Yup